charts/gatekeeper/values.yaml

```diff
@@ -10,6 +10,7 @@ auditChunkSize: 0
 logLevel: INFO
 emitAdmissionEvents: false
 emitAuditEvents: false
+mutationEnable: false
```

Comment:
I think we discussed calling this `experimental-enable-mutation` in community call and add a comment stating this flag is experimental and will change.

(this change should be in helmify)

cc @ritazh